LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -252,7 +252,42 @@ public void AssignInstructorsToClassrooms()
 
         public void RegisterStudentsForClasses()
         {
-            throw new NotImplementedException();
+            if (ClassroomDictionary.Count != 0)
```

Comment:
Add checking for active students, If you don't have active students, there is no need to continue. You can report each error separately:
if(classroomDictionary.Count == 0)
{
    // report an error : classroom count is 0
}
else if(ActiveStudentList.Count == 0) 
{
    // report an error : active student count is 0
}
else 
{
   // register students
}

Reply:
Done.